pmm-admin.go

```diff
@@ -391,6 +389,14 @@ a new user 'pmm@' automatically using the given (auto-detected) MySQL credential
   pmm-admin add mysql:metrics --password abc123 --port 3307 instance3307
   pmm-admin add mysql:queries --user rdsuser --password abc123 --host my-rds.1234567890.us-east-1.rds.amazonaws.com my-rds`,
 		Run: func(cmd *cobra.Command, args []string) {
+			// Agent does not accept additional arguments, we start it through qan-api.
```

Comment:
I don't get this comment. qan-api does not start qan-agent process.
Do you mean that we _pass_ additional arguments via qan-api.

Reply:
We start query monitoring by passing a command to qan-agent via qan-api, and neither qan-api nor qan-agent do not accept any additional arguments.